x11-misc/xscreensaver/xscreensaver-6.05.ebuild

```diff
@@ -43,21 +40,19 @@ COMMON_DEPEND="
 		x11-libs/gdk-pixbuf-xlib
```

Comment:
FWIW, the changelog for 6.0.5 reads

> X11: Removed dependence on gdk-pixbuf-xlib-2.0.

Source: https://www.jwz.org/xscreensaver/changelog.html

so this can maybe be removed?

Reply:
Yes I read that too, but configure failed for me when I unmerged that dependency. I will check again.